nexsock-web/src-ts/core/jsx-utils.ts

```
@@ -0,0 +1,177 @@
+// Simple JSX factory functions for standalone JSX without React with CSS scoping support
+
+// Forward declarations for JSX
+type CreateElementType = (
+  tag: string | Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)